lib/trackler/doc_file.rb

```diff
@@ -0,0 +1,92 @@
+class DocFile
+  DEFAULT_IMAGE_PATH = "/docs/img"
+
+  def self.find(basename:, track_dir:, image_path:)
+    dir = File.join(track_dir, "docs")
+
+    [
+      MarkdownFile.new(basename: basename, docs_dir: dir, image_path: image_path),
+      OrgmodeFile.new(basename: basename, docs_dir: dir, image_path: image_path),
+      NullFile.new(basename: basename, docs_dir: dir, image_path: image_path),
+    ].detect(&:acceptable?)
```

Comment:
It also lets you check for a more specific property rather than `acceptable`ness as the NullFile doesn't need to meet the same conditions.
`file_present?` for example.